reliability-engineering/terraform/modules/concourse-worker-pool/iam.tf

```diff
@@ -73,6 +73,7 @@ resource "aws_iam_policy" "concourse_worker_base" {
           "s3:DeleteObjectTagging",
           "s3:DeleteObjectVersion",
           "s3:DeleteObjectVersionTagging",
+          "GetBucketVersioning",
```

Comment:
It doesn't need `s3:` on the beginning?

Also https://github.com/alphagov/tech-ops/pull/49/files I believe is deployed all the way out already